extensions/applicationinsights-analytics-js/Tests/AnalyticsExtensionSize.tests.ts

```diff
@@ -0,0 +1,45 @@
+import { Assert, AITestClass } from "@microsoft/ai-test-framework";
+import * as pako from "pako";
+
+export class AnalyticsExtensionSizeCheck extends AITestClass {
+    private readonly MAX_DEFLATE_SIZE = 20;
+    private readonly filePath = "../../dist/applicationinsights-analytics-js.min.js";
```

Comment:
This one is a little different in that we should check this file AND the ../../browser/xxx version